deployment/ansible/roles/model-my-watershed.geoprocessing/templates/upstart-akka-http.conf.j2

```diff
@@ -0,0 +1,13 @@
+description "Akka HTTP Server"
+
+start on runlevel [3]
+stop on shutdown
+
+kill timeout 20
```

Comment:
This bit of mess was to play nice with the service running inside of Docker. Hopefully we can get away with just:

```
respawn
setuid mmw
chdir {{ geop_home }}
```

Need to confirm that the `mmw` user exists on this machine.